### ISSUE TITLE
Disc image file is corrupted

### ISSUE THREAD

**Issue by Sikander79** [ClusterM/hakchi2#58]
I walked through every step in order but after dumping the kernel and flashing the new custom one with new games, I connect to TV and it says "no signal". So I was worried and flashed the original kernel back on with the same results. When I open the kernel it only says "disc image file is corrupted". I am using windows 10.

I know that it isn't bricked because the computer can read the NES. I even downloaded a new kernel file from these forms and it's giving the same error message. Any help at all would be greatly appreciated and if someone has an original kernel that isn't corrupted, that would be awesome! This is frustrating because everything is so easy to do and I still need help.

Thanks in advance!

**Comment by mlabarca** [comment]
@Sikander79 Did you manage to solve this? any issues installing the driver? Can you send steps and configuration you synced to be able to reproduce?

**Comment by Sikander79** [comment]
Thank you for your reply. I have not solved this as of yet. I followed the
instructions completely accept that I did not back up original kernel
because I am not very knowledgeable with programming and configuration and
did not know what a kernel was. I do now.

Driver installed without any problems.

When I first began I connected NES to my computer and held down the reset
button and then pushed the power button, I waited 5 to 10 seconds and let
go of reset button. I already had my ROMS in place to load onto NES. I
dumped original kernel (but did not back up) and flashed the custom kernel.
After I synced my ROMS to NES it said that I was done and to restart my
NES. I depressed power button and connect NES to TV and turned the power on
but got nothing but a blank screen saying "No Signal". I knew something was
very wrong so I connected once again to the computer and flashed
the original kernel and once done I reconnected it to the TV but with the
same results.

It is not bricked but it doesn't want to accept a new kernel or custom one.
I don't know what to do?

Every time I download the original kernel from the forum and open it, it
says "DISC IMAGE FILE IS CORRUPTED". I am running windows 10. I do have a
anti-malware and anti virus program on my computer, but I have configured
my settings to allow me to open files from unknown sources.

I am sorry that I do not know how to send you the configuration I synced.

I would LOVE to get this working even if it only means to get back my
original games. I have dumped and flashed the original kernel and the
custom several times trying to follow each step again and again. This is a
fantastic program, I just wish I could get a non-corrupted kernel or that
my computer would allow me to open the ones that I've tried. I HATE windows
10.

Thank you,

TJ

**Comment by mlabarca** [comment]
"After I synced my ROMS to NES it said that I was done and to restart my
NES. I depressed power button and connect NES to TV..." If I remember
correctly there was a step where you had to wait for the power led to go
off before resetting it and connecting to the tv. (Anyone else can
confirm?) Were you using the most recent version btw?
I used win 10 and every thing went without a hitch after I managed to
install the driver, but after hearing your story I don't think i'll flash
it again with more roms or recommend this tool after reading this.

**Comment by Sikander79** [comment]
I waited for the light to go off and I am using the latest version. Where can I get another original kernel?

**Comment by Sikander79** [comment]
This problem has been solved. Something within my Windows 10 programming blocked or corrupted the kernel each time I opened it. I still have not figured out the cause of this.

I did use another laptop from a buddy with Windows 7 and had no trouble at all! It was easy, straightforward, and an excellent tool for NES.The problems were on MY end. 

The Hakchi2 is very well done. Thanks for all of the help!
